tests/phpunit/TemplateTest.php

```diff
@@ -1053,6 +1053,9 @@ public function testTrimResearchGateETC() {
     $text = '{{cite journal|url=http://researchgate.net/publication/320041870_EXTRA_STUFF_ON_END}}';
     $prepared = $this->prepare_citation($text);
     $this->assertEquals($want, $prepared->get('url'));
+    $text = '{{cite journal|url=https://www.researchgate.net/profile/hello_user-person/publication/320041870_EXTRA_STUFF_ON_EN}}';
```

Comment:
Do we need both tests here, or can we simply replace the previous test with the new one?

Reply:
I want to test both just to make sure we don’t accidentally require a / character.   I almost did that error